content/install-hook-userstyles.js

```diff
@@ -169,7 +171,8 @@
           'https://github.com/openstyles/stylus/issues/195');
         return;
       }
-      return API.installStyle(Object.assign(json, addProps))
+      // Update originalMd5 since USO changed it (2018-11-11) to NOT match the current md5
+      return API.installStyle(Object.assign(json, addProps, {originalMd5: currentMd5}))
```

Comment:
We may have to ensure `currentMd5` is set before calling `API.installStyle`.

Reply:
I'm not sure it still needs this since `currentMd5` is sure to be set now.